### DISCUSSION TITLE
Move drawn items under chat hud

### DISCUSSION THREAD

**Discussion by legoraft** [FabricMC/fabric-api#1742] [Mod Dev Support]
For a mod I'm making, I'm drawing some armor items onto the HUD, the problem is that it also overlays the [chat](https://user-images.githubusercontent.com/50689727/131455130-9d09472b-81d8-49bc-b662-913492b20e60.png).

The code I used is this (This is injected into the renderHotbar function in the InGameHud.class):
```java
this.renderHotbarItem(this.scaledWidth / 2 + 18, i, tickDelta, client.player, this.client.player.getInventory().getArmorStack(3), 1);
this.renderHotbarItem(this.scaledWidth / 2 + 33, i, tickDelta, client.player, this.client.player.getInventory().getArmorStack(2), 1);
this.renderHotbarItem(this.scaledWidth / 2 + 48, i, tickDelta, client.player, this.client.player.getInventory().getArmorStack(1), 1);
this.renderHotbarItem(this.scaledWidth / 2 + 63, i, tickDelta, client.player, this.client.player.getInventory().getArmorStack(0), 1);
```

Does anyone know how to get the armor items under the chat hud?

**Comment by Technici4n** [comment]
Try HudRenderCallback perhaps?

**Reply by legoraft** [reply]
Would work, but I'm not using the fabric API, so wouldn't work for me, thanks anyways!

**Reply by Technici4n** [reply]
Why do you ask the question here then? Also you could look at where the HudRenderCallback is injected.

**Reply by legoraft** [reply]
thought I could ask general fabric advice, not only fabric api solutions. Will take a look at the HudRenderCallback inject
